pages/dashboard/projects.vue

```diff
@@ -319,179 +319,146 @@ import SaveIcon from '~/assets/images/utils/save.svg'
-          discord_url: this.editLinks.discord.clear ? null : this.editLinks.discord.val.trim(),
-        }
+function updateDescending() {
+  descending.value = !descending.value
+  projects.value = updateSort(projects.value, sortBy.value, descending.value)
+}
 
-        if (!baseData.issues_url?.length ?? 1 > 0) {
-          delete baseData.issues_url
+function updateSort(projects, sort, descending) {
+  let sortedArray = projects
+  switch (sort) {
+    case 'Name':
+      sortedArray = projects.slice().sort((a, b) => {
+        return a.title.localeCompare(b.title)
+      })
+      break
+    case 'Status':
+      sortedArray = projects.slice().sort((a, b) => {
+        if (a.status < b.status) {
+          return -1
         }
-
-        if (!baseData.source_url?.length ?? 1 > 0) {
-          delete baseData.source_url
+        if (a.status > b.status) {
+          return 1
         }
-
-        if (!baseData.wiki_url?.length ?? 1 > 0) {
-          delete baseData.wiki_url
+        return 0
+      })
+      break
+    case 'Type':
+      sortedArray = projects.slice().sort((a, b) => {
+        if (a.project_type < b.project_type) {
+          return -1
         }
-
-        if (!baseData.discord_url?.length ?? 1 > 0) {
-          delete baseData.discord_url
+        if (a.project_type > b.project_type) {
+          return 1
         }
+        return 0
+      })
+      break
+    default:
+      break
+  }
 
-        await useBaseFetch(
-          `projects?ids=${JSON.stringify(this.selectedProjects.map((x) => x.id))}`,
-          {
-            method: 'PATCH',
-            body: baseData,
-          }
-        )
-
-        this.$refs.editLinksModal.hide()
-        this.$notify({
-          group: 'main',
-          title: 'Success',
-          text: "Bulk edited selected project's links.",
-          type: 'success',
-        })
-        this.selectedProjects = []
-
-        this.editLinks.issues.val = ''
-        this.editLinks.source.val = ''
-        this.editLinks.wiki.val = ''
-        this.editLinks.discord.val = ''
-        this.editLinks.issues.clear = false
-        this.editLinks.source.clear = false
-        this.editLinks.wiki.clear = false
-        this.editLinks.discord.clear = false
-      } catch (e) {
-        this.$notify({
-          group: 'main',
-          title: 'An error occurred',
-          text: e,
-          type: 'error',
-        })
-      }
-    },
-  },
-})
+  if (descending) {
+    sortedArray = sortedArray.reverse()
+  }
+
+  return sortedArray
+}
+
+async function bulkEditLinks() {
+  try {
+    const baseData = {
+      issues_url: editLinks.value.issues.clear ? null : editLinks.value.issues.val.trim(),
+      source_url: editLinks.value.source.clear ? null : editLinks.value.source.val.trim(),
+      wiki_url: editLinks.value.wiki.clear ? null : editLinks.value.wiki.val.trim(),
+      discord_url: editLinks.value.discord.clear ? null : editLinks.value.discord.val.trim(),
+    }
```

Comment:
Don't forget to update other references to `editLinks` now that it is a ‘plain’ object
```suggestion
    const baseData = {
      issues_url: editLinks.issues.clear ? null : editLinks.issues.val.trim(),
      source_url: editLinks.source.clear ? null : editLinks.source.val.trim(),
      wiki_url: editLinks.wiki.clear ? null : editLinks.wiki.val.trim(),
      discord_url: editLinks.discord.clear ? null : editLinks.discord.val.trim(),
    }
```

Reply:
Fixed ^^.